File: Programs/5 Languages/HW01.scala
object HW01 {
   def main(args: Array[String]) {
      println("Hello, World!") 

      println("Input a number: ") 
      var x = Console.readInt;
      var nCnt = 0;
      var i = 0;
      println("Your entered number: " + x);

      for(i <- 1 to x) {
      	if(x % i == 0) {
      		nCnt += 1;
      	}
      }

      if(nCnt > 2) {
      	println("This number is not prime.");
      } else {
      	println("This number is prime.");
      }

      var result = (x * x)  + ((scala.math.log10(x) - scala.math.sin(x))/scala.math.sqrt(x));
      println(result);

      var hi = 5;
      var hello = 6.5;
      hi = hello;

      println(hi);
   }
}

File: Programs/5 Languages/HW01.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HW01 {
    class Program {
        static void Main(string[] args) {
            Console.WriteLine("Hello, World");

            Console.Write("Input a number: ");
            int x = Convert.ToInt32(Console.ReadLine());
            int nCnt = 0;
            Console.WriteLine("Your entered number: " + x);

            for(int i = 1; i < x; i ++) {
                if(x % i == 0) {
                    nCnt++;
                }
            }
            
            if(nCnt > 2) {
                Console.WriteLine("Number is not prime.");
            } else {
                Console.WriteLine("Number is prime.");
            }

            double result = (x * x + ((Math.Log10(x) - Math.Sin(x)))/Math.Sqrt(x));
            Console.WriteLine(result);
            while (true) ;
        }
    }
}
